src/components/FooterLogo.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import get from 'lodash/get'
+import { withClient } from 'cozy-client'
+
+export class FooterLogo extends React.Component {
+  state = {
+    logos: []
+  }
+
+  async componentDidMount() {
+    const context = await this.props.client
+      .getStackClient()
```

Comment:
I always use directly `this.props.client.stackClient`, I think we should have `getStackClient` as a getter.